flink-state-backends/flink-statebackend-forst/src/main/java/org/apache/flink/state/forst/snapshot/ForStNativeFullSnapshotStrategy.java

```diff
@@ -203,6 +205,7 @@ public ForStNativeSnapshotResources syncPrepareResources(long checkpointId) thro
                     backendUID,
                     checkpointId);
             db.enableFileDeletions(false);
+            lease.close();
```

Comment:
maybe move to a finally clause?

Reply:
No, because we should close `lease` in `releaser`, and the `releaser` may be invoked after `finally clause`.